firmware/CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ Release template (copy/paste this for new release):
  -   
 
 ### Fixed
- -   
+ - Insist on setBoardConfigOverrides #4614   
```

Comment:
maybe human description, not opaque issue title?

"enforce board configuration overrides more strictly"